lib/ansible/modules/database/postgresql/postgresql_facts.py

```diff
@@ -0,0 +1,916 @@
+                    returned: always
+                    type: str
+                    sample: "{postgres=UC/postgres,=U/postgres}"
+                lanowner:
+                    description:
+                      - Language owner U(https://www.postgresql.org/docs/current/catalog-pg-language.html).
+                    returned: always
+                    type: str
+                    sample: postgres
+        namespaces:
+            description:
+              - Namespaces (schema) U(https://www.postgresql.org/docs/current/sql-createschema.html).
+            returned: always
+            type: dict
+            sample: pg_catalog
+            contains:
+                nspacl:
+                    description:
+                      - Assess privileges U(https://www.postgresql.org/docs/current/catalog-pg-namespace.html).
+                    returned: always
+                    type: str
+                    sample: "{postgres=UC/postgres,=U/postgres}"
+                nspowner:
+                    description:
+                      - Schema owner U(https://www.postgresql.org/docs/current/catalog-pg-namespace.html).
+                    returned: always
+                    type: str
+                    sample: postgres
+        repl_slots:
+            description:
+              - Replication slots (available in 9.4 and later)
+                U(https://www.postgresql.org/docs/current/catalog-pg-replication-slots.html).
+            returned: if existent
+            type: dict
+            sample: slot0
+            contains:
+                active:
+                    description: True if this slot is currently actively being used.
+                    returned: always
+                    type: bool
+                    sample: true
+                database:
+                    description: Database name this slot is associated with, or null.
+                    returned: always
+                    type: str
+                    sample: acme
+                plugin:
+                    description:
+                      - The base name of the shared object containing the the output plugin
+                        this logical slot is using, or null for physical slots.
+                    returned: always
+                    type: str
+                    sample: pgoutput
+                slot_type:
+                    description: The slot type - physical or logical.
+                    returned: always
+                    type: str
+                    sample: logical
+        replications:
+            description:
+              - Information about the current replications by process PIDs
+                U(https://www.postgresql.org/docs/current/monitoring-stats.html#MONITORING-STATS-VIEWS-TABLE).
+            returned: if pg_stat_replication view existent
+            type: dict
+            sample: 47823
+            contains:
+                usename:
```

Comment:
Should this not be **username** ?

Reply:
This is because in the source system table pg_stat_replication, this column named "usename". I tried to name parameters in accordance to their names in postgres statistics tables.
Well.. I was thinking about it too. The question is what's better in Ansible case... maybe this module will be used not only DBAs. I should think. Maybe would be better to name it like "useRname". I don't really understand why postgresql developers somewhere named similar column in system tables/views like "rolname", "rolename", "usename", etc. ;)

Reply:
According to my explanation above, give me your opinion, in this case (Ansible) **username** is better?

Reply:
I'll changed because it looks like a type ;)

Reply:
If it is a low-level thing, by all means keep it at `usename` !